dnf/fapolicyd-dnf-plugin.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+
+import dnf
+import os
+import stat
+import signal
+
+class Fapolicyd(dnf.Plugin):
+
+    name = "fapolicyd"
+    pipe = "/var/lib/fapolicyd/fapolicyd.fifo"
```

Comment:
Reviewing the FHS specification, /var/lib is to preserve information between runs of the application. For example, libvirt images are in the directory because they need to persist between runs. For things that live since boot, its suggested to put things in /var/run. Looking in there, I see libvirt places its sockets there. I'd suggest that we switch directories.

Reply:
Yes, that would be better.